### ISSUE TITLE
Remaining time to download is negative

### ISSUE THREAD

**Issue by ppisar** [rpm-software-management/dnf5#1963]
A dowload progressbar reports time remaining to finishing the download. An example:

~~~~
# dnf --setopt=optional_metadata_types=filelists makecache --refresh
Updating and loading repositories:
 Fedora 42 on Koji                          1% [                  ] |   4.9 KiB/s | 854.1 KiB | -03h18m
~~~~

Observe negative value "-03h18m" value in the last column. In my opinion, the time should be counted from a positive value to zero.

Downstream bug: https://bugzilla.redhat.com/show_bug.cgi?id=2332931

**Comment by evan-goode** [comment]
I agree that negative remaining time is unidiomatic and we should change it. It was implemented intentionally:

https://github.com/rpm-software-management/dnf5/blob/d51f61461d223de9c347f72a2c373852c1df7a56/libdnf5-cli/progressbar/widgets/common.cpp#L42-L44

I filed https://github.com/rpm-software-management/dnf5/pull/1971. `format_time(..., negative=true)` does not seem to occur anywhere else.
